### DISCUSSION TITLE
Help with filters [Please Help]

### DISCUSSION THREAD

**Discussion by usamamumtaz1** [Budibase/budibase#6258] [Help] [answered]
Hello guys

i m sorry if this has been asked already but  i would extremely appreciate your help here. 

i m creating a simple app with two table:

Table 1: Mentor
1- Name
2- Email
Table 2: Mentee
1- Name
2- Email
3- Mentor (relationship field with table 1.Name)

i want to create a table with all Mentee information appearing with a filter of Mentor but when Option picker with Mentor, i m getting this:
![2022-06-09 at 21 55](https://user-images.githubusercontent.com/96530344/172942796-a6ef0969-bc1f-4189-8ae3-1c10eb4a3e30.png)

i even tried to add two separate data provider, one for Mentor and other one with Mentee but when i try to define filter, i cant find the Mentor field there. 

![2022-06-09 at 21 55](https://user-images.githubusercontent.com/96530344/172942904-4c9df6d6-a779-4bee-bfeb-26162c369f46.png)

its such a basic functionality i was hoping to build here but cant seems to be working here. 

can you help me with this?

Thanks

**Comment by melohagan** [comment, marked as answer]
Hi @usamamumtaz1 

If you want to select relationships, you need to use the *Relationship Picker* component, and not the *Options Picker*.

It should be noted that unfortunately you cannot filter on relationships when using the Internal DB.

**Reply by usamamumtaz1** [reply]
Thank you so much!!
